README-SUBMODULES.md

```diff
@@ -0,0 +1,71 @@
+# How to convert Mercurial Repositories with subrepos
+
+## Introduction
+
+First a mapping file needs to be created that maps between the subrepo folder
+and the corresponding git submodule. The format for this file is:
+
+:<mercurial subrepo folder> <git submodule name>|<SKIP>
+:<mercurial subrepo folder2> <git submodule name2>|<SKIP>
+...
+
+The "SKIP" keyword can be used to skip submodules that might not be available any
+longer.
+
+A typical example of this file looks like
+:subrepo/subrepo1 ../submodule1
+:subrepo/subrepo2 ../submodule2
+:subrepo/subrepo3 SKIP
+
+The created .gitmodules file would look the look like
+
+    [submodule "subrepo/subrepo1"]
+          path = subrepo/subrepo1
+          url = ../submodule1
+
+    [submodule "subrepo/subrepo2"]
+          path = subrepo/subrepo2
+          url = ../submodule2
+
+## Example
```

Comment:
This example section should go first, before the mapping and .gitmodules examples. That way the reader won't have to guess about the relation between the examples and the hg repo.

Reply:
Since the example here already contains an example mapping file I will simply remove the above mapping example and move down the .gitmodules example result.